lib/cryptoexchange/exchanges/novaexchange/models/market_pair.rb

```diff
@@ -0,0 +1,14 @@
+module Cryptoexchange::Exchanges
+  module Novaexchange
+    module Models
+      class MarketPair < Cryptoexchange::Models::MarketPair
+        attr_accessor :name
+
+        def initialize(params={})
+          super
+          @name = params[:name]
```

Comment:
This is used to query ticker for individual trading pair (see [here][ticker-url]). For most (but not all) trading pairs, the name is just `#{base}_#{target}`. So I'm using this to keep track of its name.

[ticker-url]: https://github.com/yihangho/cryptoexchange/blob/777720db00a4a584d1e933aff3044cd63740483b/lib/cryptoexchange/exchanges/novaexchange/services/market.rb#L19